### PR TITLE
Add themes for remaining buttons

### LINKED ISSUE #759: Dark theme: Next, Done and Done & Talk dark theme styles
**Package** lib-grommet-theme, lib-classifier

@rogerhutchings I'm assigning you since you've done the dark theme work most recently.

**Feature or Issue Description**
The dark theme styles mean switching to a outline border style for the task navigation and slight color changes:

Screen Shot 2019-04-24 at 10 08 02 AM
Screen Shot 2019-04-24 at 10 08 38 AM

How they appear in the new app:

Screen Shot 2019-04-24 at 10 09 27 AM
Screen Shot 2019-04-24 at 10 09 51 AM

@beckyrother any changes here or should we replicate the current implementation?

### PR DESCRIPTION
**Pull request by rogerhutchings**
Closes #759, #739

Adds themes for the remaining `TaskArea` buttons. There's probably some code that can be abstracted out, but that can wait for another PR.

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?
- [ ] Is the changelog updated?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?

### CODE REVIEW

File: packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components/DoneAndTalkButton/DoneAndTalkButton.spec.js [hunk 1]
```diff
@@ -28,16 +29,4 @@ describe('DoneAndTalkButton', function () {
       expect(wrapper.html()).to.be.null
     })
   })
-
-  describe('when props.completed is false', function () {
```
Comment (eatyourgreens) [inline packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components/DoneAndTalkButton/DoneAndTalkButton.spec.js]: Has props.completed been removed or are we just ignoring the false case in the tests now?

Comment (srallen) [reply packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components/DoneAndTalkButton/DoneAndTalkButton.spec.js]: In PFE, completed is a boolean being set by task completion validation functions per task type. None of that code has moved over and I'm not sure if we want to reuse any of it.

**Comment by srallen** [comment]
I've set the defaults for padding for buttons in the theme in #784 since the default should be the 40px height. If you merge that first, then you should be able to remove the customization in the theme extensions here for the paddings.

**Comment by rogerhutchings** [comment]
Sorry, I didn't realise we were using the old PFE colours

**Comment by rogerhutchings** [comment]
Without the original padding settings, the buttons come out too short at 38px; I'm adding them back in.

**Comment by srallen** [comment]
@rogerhutchings hm, maybe it's because it's the button part of the theme object being deep merged so our theme's default setting got overwritten back to the Grommet default? I wouldn't have expected that.

**Comment by srallen** [comment]
> I didn't realise we were using the old PFE colours, there are some hover states in InVision and I was using those.

@beckyrother could you clarify which colors should be used for the Done & Talk and Done button's on hover for the dark theme? Thanks!

**Comment by rogerhutchings** [comment]
(hex codes would be nice)

**Comment by srallen** [comment]
I've been using this site to figure out which function to use if the color is a variant of one of the theme bases: https://www.ofcodeandcolor.com/cuttle/ (the functions suggested are sass/less but almost always map to something available in polished.js)

**Comment by srallen** [comment]
FWIW, I remember we did a revision on the colors of the dark mode after volunteer feedback and made some changes of the colors to increase contrast. It's possible those updates didn't make it to invision. I can try to find the PRs on PFE as a ref.

**Comment by beckyrother** [comment]
That's right, the original colors didn't have enough contrast so we updated on PFE. I'm still not 100% happy with them though so I will post new hex colors and update InVision by EOD today.

**Comment by rogerhutchings** [comment]
@srallen that link is awesome <3

**Comment by beckyrother** [comment]
[Updated the InVision](https://invis.io/UAN6AP0JTBC#/288027570_08-3_Dark_Finished) with some more thoughtful styles. I'll also update the 'styleguide' that's in the InVision assets, but here's what it should look like. Please note the subtle shadows that help the outlines stand out from the background (apologies if the syntax is incorrect). 

Nothing selected: 
image
`Done: #078F52; box-shadow: 1px 1px 2px rgba(0, 0, 0, .5);`
`Done & Talk border: 1px solid #43BBFD; box-shadow: 1px 1px 2px rgba(0, 0, 0, .5);`

Done hover:
image
`Done: #034628; border: 1px solid #B8E986; box-shadow: 1px 1px 2px rgba(0, 0, 0, .5);`

Done & Talk hover:
image
`Done & Talk hover: #104A79; border: 1px solid #43BBFD; box-shadow: 1px 1px 2px rgba(0, 0, 0, .5);`

**Comment by beckyrother** [comment]
_Also this_

Hover on answer (removed the gradient):
image
`button hover: #004954; border: 1px solid #EFF2F5;`

Selected answer:
image
`button selected: #00979d; border: 1px solid #EFF2F5;`

File: packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components/DoneButton/theme.js [hunk 2]
```diff
@@ -24,7 +22,7 @@ const theme = {
 
       &:focus:not(:disabled),
       &:hover:not(:disabled) {
-        background: ${DARK_GREEN};
+        background: ${darken(0.15, props.theme.global.colors['status-ok'])};
         box-shadow: none;
```
Comment (srallen) [inline packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components/DoneButton/theme.js]: `box-shadow: none;` should be removed

Comment (rogerhutchings) [reply packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components/DoneButton/theme.js]: I remember why I did this - the default Grommet button theme uses `box-shadow` to create an extra border on focus / hover

**Comment by srallen** [comment]
I'll note style updates for the radio buttons in a separate issue. This is looking good for the Done and Done & Talk.

**Comment by beckyrother** [comment]
Just mentioning again, Jim brought up in #design-thoughts that we may want to consider a different visual style for multi-select vs radio buttons for screen readers and voice control. When we get to the survey task let's talk about it more.

**Pull request merged**